### ISSUE TITLE
Exception when presenting `EKMessageContentView` iOS 11

### ISSUE THREAD

**Issue by markst** [huri000/SwiftEntryKit#253]
Crashes when `setupInitialPosition()` is called & `contentView` is added to `EKContentView`.

![image](https://user-images.githubusercontent.com/274318/68095616-fd84d500-fefe-11e9-8de0-8c33a3b829ac.png)

**Comment by markst** [comment]
```swift
  func presentPopupMini(with message: String, style: PopupStyle = .blue) {
    var attributes = EKAttributes.bottomToast
    attributes.roundCorners = .top(radius: style.cornerRadius)
    attributes.entryBackground = .color(color: .init(style.backgroundColour))
    attributes.positionConstraints.size = .init(width: .fill, height: .intrinsic)
    attributes.displayDuration = .infinity

    let messageContent = EKMessageContentView()
    messageContent.titleContent = .init(text: message,
                                        style: .init(font: .nunitoBold(style.fontSize),
                                                     color: .init(style.textColour),
                                                     alignment: .center,
                                                     displayMode: .inferred,
                                                     numberOfLines: 0))

    SwiftEntryKit.display(entry: messageContent, using: attributes)
  }
```

**Comment by huri000** [comment]
Hi @markst, thanks for providing the example. A fix is available on `1.2.3` 🎉
